### DISCUSSION TITLE
Is there a peek function and a transparency function?

### DISCUSSION THREAD

**Discussion by weirdalsuperfan** [Ross-Patterson/Portals-Desktop-Organization#167] [Q&A] [answered]
I don't want to use Fences 4 because apparently it doesn't let you organize your icons within the fences.

However, Fences 4 DOES allow you to peek at your desktop without minimizing your screen, which is what I'm looking for.

I don't really care about any other features - I don't need to add labels to the different sections on my desktop where I place icons.

I don't like minimizing my open windows to open a program, because as soon as a new program opens, Windows forgets what all I had open, so I have to click the Show Desktop button super fast after double-clicking what I want to open, which is often impossible. So that's why I want a peek feature.

So I'm wondering 2 things:
1. Does Portals have such a peek feature?

2. Also, since I don't like seeing the fences (I'd rather see my Desktop background), I'm looking for full-transparency of the portals/fences. Does Portals allow this?

**Comment by Ross-Patterson** [comment, marked as answer]
1. Yes, you can turn it on in the settings.

2. Not FULL transparency, because then the window becomes non-clickable (Windows behaviour that cannot be changed), but you can make it like 99.9% transparent which is barely noticeble.

**Comment by stevemagruder** [comment]
Does the peek feature work under Windows 11 24H2? It broke in Fences 4. If it works in Portals, that will prompt me to use Portals instead.
